src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -36,24 +36,20 @@ private void readObject(java.io.ObjectInputStream in)
     @Argument(fullName = StandardArgumentDefinitions.READ_FILTER_LONG_NAME,
             shortName = StandardArgumentDefinitions.READ_FILTER_SHORT_NAME,
             doc="Read filters to be applied before analysis", optional=true, common = true)
-    public final List<String> userReadFilterNames = new ArrayList<>(); // preserve order
+    public final List<String> userEnabledReadFilterNames = new ArrayList<>(); // preserve order
 
     @Argument(fullName = StandardArgumentDefinitions.DISABLE_READ_FILTER_LONG_NAME,
             shortName = StandardArgumentDefinitions.DISABLE_READ_FILTER_SHORT_NAME,
             doc="Read filters to be disabled before analysis", optional=true, common = true)
-    public final List<String> disableFilters = new ArrayList<>();
+    public final List<String> userDisabledReadFilterNames = new ArrayList<>();
 
     @Argument(fullName = StandardArgumentDefinitions.DISABLE_TOOL_DEFAULT_READ_FILTERS,
             shortName = StandardArgumentDefinitions.DISABLE_TOOL_DEFAULT_READ_FILTERS,
             doc = "Disable all tool default read filters", common = true, optional = true)
     public boolean disableToolDefaultReadFilters = false;
 
     // Map of read filter (simple) class names to the corresponding discovered plugin instance
-    private Map<String, ReadFilter> readFilters = new HashMap<>();
-    // readFilters map is overridden in validateArguments
-    // this set is for keeping the names for the help in alphabetical order
-    // and to use them for checking if an argument exists or not
-    private final Set<String> readFilterNames = new TreeSet<>();
+    private Map<String, ReadFilter> allDiscoveredReadFilters = new HashMap<>();
```

Comment:
What do you think if we make all these fileds "final" to avoid overriding as previously, @cmnbroad?